### PR TITLE
Space Travelers Hub 

### PR DESCRIPTION
**Pull request by Debas-31**
# 👋 Micronaut Reviewer 
- [x] Use React documentation.
- [x] Use React components.
- [x] Use React props.
- [x] Use React Router.
- [x] Connect React and Redux.
- [x] Handle events in a React app.
- [x] Write unit tests with React Testing Library.
- [x] Use styles in a React app.
- [x] Use React life cycle methods.
- [x] Apply React best practices and language style guides in code.
- [x] Use store, actions, and reducers in React.
- [x] Perform a code review for a team member.

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (ZeenatLawal) [inline public/manifest.json:1]: - [ ] All boilerplate files and icons should be deleted as they are not required for the project. e.g In the public folder(the logo images, manifest.json and robots.txt).

File: src/__tests__/App.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```
Comment (ZeenatLawal) [inline src/__tests__/App.test.js:1]: - [ ] Kindly write more unit tests for the other components using React Testing Library and Jest.

File: src/components/navbar/Navbar.module.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+.logo {
+  width: 3rem;
+  height: 3rem;
```
Comment (ZeenatLawal) [inline src/components/navbar/Navbar.module.css:3]: - [ ] Kindly adjust the height of the Navbar logo to match the given design, the logo looks a bit squeezed.
FireShot Capture 170 - React App - localhost

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Debas-31 and @akucintavalent, 

Good job so far!:clap: :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎊 🎊 
- Git flow is followed. ✔️
- PR has a title and summary. ✔️


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/navbar/Navbar.module.css (hunk 3 above)
Comment (akucintavalent) [reply src/components/navbar/Navbar.module.css:3]: May I ask what browser you are using? @ZeenatLawal

Comment (ZeenatLawal) [reply src/components/navbar/Navbar.module.css:3]: @akucintavalent I use chrome

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Debas-31 and @akucintavalent, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**